com.Acestream.engine.appdata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.Acestream.engine</id>
+  <name>Acestream</name>
+<developer_name>Acestream-engine-flatpak</developer_name>
+<project_license>LicenseRef-proprietary</project_license>
+<metadata_license>CC0-1.0</metadata_license>
+<summary>Ace Stream</summary>
+<url type="homepage">https://forum.acestream.media</url>
+<description><p>Ace Stream is a media platform of a new generation with a decentralized system of delivering and storing multimedia data,</p><p> which is designed to create Internet services with functions of AVoD (Audio and Video on Demand) and Live Streaming,</p><p> with extensive social and interactive component.</p><p> If you are using the GNOME environment, you need to install extensions.gnome.org/extension/1503/tray-icons to work with the application</p></description>
+ <releases>
+    <release date="2020-7-22" version="0.2"/> 
```

Comment:
These versions do not seem to match what manifest is downloading.

Reply:
did not change for the test whether the version check will work (x-checker-data) (403)